### PR TITLE
chore: Migrate from gerrit to github pipelines

### PR DESCRIPTION
**Pull request by Rolika4**
# Pull Request Template

## Description
Please include a summary of the change and why it is needed.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/e2e/capsule-feature/02-install-operator.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - command: |
-      helm install --set image.repository=${E2E_IMAGE_REPOSITORY}
+      helm install --set image.repository=${CONTAINER_REGISTRY_URL}/${CONTAINER_REGISTRY_SPACE}/${E2E_IMAGE_REPOSITORY}
```
Comment (zmotso) [inline tests/e2e/capsule-feature/02-install-operator.yaml:5]: CONTAINER_REGISTRY_URL, CONTAINER_REGISTRY_SPACE - how do you pass this to CI?

Comment (zmotso) [reply tests/e2e/capsule-feature/02-install-operator.yaml:5]: We don't need to update the `make e2e` command?

Comment (Rolika4) [reply tests/e2e/capsule-feature/02-install-operator.yaml:5]: already updated, wait for check it on github actions

**Review by zmotso — APPROVED** [review]

File: tests/e2e/capsule-feature/02-install-operator.yaml (hunk 1 above)
Comment (Rolika4) [reply tests/e2e/capsule-feature/02-install-operator.yaml:5]: fixed

**Pull request merged**